app/assets/stylesheets/comments.scss

```diff
@@ -851,9 +873,13 @@ a.header-link {
       overflow-x: auto;
       code {
         background: #29292e;
-        color: #eff0f9;
+        color: black;
         white-space: pre;
         font-size: 1em;
+        background-color: white;
+        padding-top: 0%;
```

Comment:
Skip the `%` here since its not needed.

Reply:
Removed `%` in [this](https://github.com/thepracticaldev/dev.to/pull/7103/commits/af9233a10f446ca71a71b00fbabb4e7531de2e5a#diff-53e600e3c9b5ce6a6226b7245ec587f3R885) commit.